### PR TITLE
Feat/trending section

### PR DESCRIPTION
**Pull request by Isaiahpkelly**
added the trending section.

### CODE REVIEW

**Review by katjones23 — CHANGES_REQUESTED** [review]
Script tag should be at the bottom of the body.

File: index.html [hunk 1]
```diff
@@ -190,38 +177,7 @@ <h1>Trending Now</h1>
...
-            <li><a class="grey-text text-lighten-3" href="#!">Link 4</a></li>
-          </ul>
-        </div>
-      </div>
-    </div>
 
-  </footer>
 </body>
 
 <script src="./index.js"></script>
```
Comment (katjones23) [inline index.html]: Script at the bottom of the body

**Review by katjones23 — APPROVED** [review]

**Review by tmanderson001 — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -55,15 +53,134 @@
...
+        </div>
+      </div>
+    </section>
+  </div>
+  </section>
   
+  <script src="./index.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
+  <script src="https://code.jquery.com/jquery-3.4.1.min.js"
+    integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
```
Comment (katjones23) [inline index.html:182]: Jquery should be the first script, then Materialize CDN, finally our JS file

File: index.js [hunk 3]
```diff
@@ -1,8 +1,25 @@
 
   $(document).ready(function(){
     $('.sidenav').sidenav();
+    alert('hi');
```
Comment (katjones23) [inline index.js:4]: If testing is done, should remove this alert

File: index.js [hunk 4]
```diff
@@ -1,8 +1,25 @@
...
+   
   });
- 
- 
+
+  $(".searchBtn").on("click", function(event) {
+    // event.preventDefault() prevents the form from trying to submit itself.
+    // We're using a form so that the user can hit enter instead of clicking the button if they want
+    event.preventDefault();
+    alert('hi');
```
Comment (katjones23) [inline index.js:12]: If testing is done, should remove this alert

**Review by katjones23 — CHANGES_REQUESTED** [review]
please see comments

File: index.html (hunk 2 above)
Comment (Isaiahpkelly) [reply index.html:182]: Hey Katherine, I think we have been and fourth on this one. I have changed it both ways but believe we came to an agreement with Esterling on this way
Screenshot (134)

File: index.js (hunk 3 above)
Comment (Isaiahpkelly) [reply index.js:4]: Will make sure I remove it going forward. It seems as if everything is already changed when I reopen the branch

**Pull request closed without merging**